Formula/ooniprobe.rb

```diff
@@ -15,275 +15,108 @@ class Ooniprobe < Formula
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/b3/b2/238e2590826bfdd113244a40d9d3eb26918bd798fc187e2360a8367068db/six-1.10.0.tar.gz"
-    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
-  end
-
-  resource "Twisted" do
-    url "https://files.pythonhosted.org/packages/d2/5d/ed5071740be94da625535f4333793d6fd238f9012f0fee189d0c5d00bd74/Twisted-17.1.0.tar.bz2"
-    sha256 "dbf211d70afe5b4442e3933ff01859533eba9f13d8b3e2e1b97dc2125e2d44dc"
-  end
-
-  resource "txsocksx" do
-    url "https://files.pythonhosted.org/packages/ed/36/5bc796eb2612b500baa26a68481d699e08af5382152a9de18e5a45b44ea7/txsocksx-1.15.0.2.tar.gz"
-    sha256 "4f79b5225ce29709bfcee45e6f726e65b70fd6f1399d1898e54303dbd6f8065f"
-  end
-
-  resource "txtorcon" do
-    url "https://files.pythonhosted.org/packages/03/23/4453ab8728c84963cf293d5180b9f529bb10bf8285031c4681e2621b175f/txtorcon-0.18.0.tar.gz"
-    sha256 "12be80f1d5e2893378c6e8c752cf159479f868f8424e16b34b75cd679a0ab171"
-  end
-
-  resource "Werkzeug" do
-    url "https://files.pythonhosted.org/packages/13/a2/c4f2a1e1e9239cd979de00a2d7e0008559c46d920e9842e9b8063c5e6bf5/Werkzeug-0.12.tar.gz"
-    sha256 "f007848ed997101cb5c09a47e46c0b0b6f193d0f8a01cd2af920d77bf1ab4e68"
-  end
-
-  resource "zope.interface" do
-    url "https://files.pythonhosted.org/packages/44/af/cea1e18bc0d3be0e0824762d3236f0e61088eeed75287e7b854d65ec9916/zope.interface-4.3.3.tar.gz"
-    sha256 "8780ef68ca8c3fe1abb30c058a59015129d6e04a6b02c2e56b9c7de6078dfa88"
-  end
+  depends_on "go@1.14" => :build
 
   def install
-    # provided by libdnet
-    inreplace "requirements.txt", "pydumbnet", ""
-
-    # force a distutils install
-    inreplace "setup.py", "def run(", "def norun("
-
-    # obey the settings.ini we write
-    inreplace "ooni/settings.py", /(IS_VIRTUALENV = ).*/, "\\1 False"
-
-    (buildpath/"ooni/settings.ini").atomic_write <<~EOS
-      [directories]
-      usr_share = #{pkgshare}
-      var_lib = #{var}/lib/ooni
-      etc = #{etc}/ooni
-    EOS
-
-    ENV.append "CPPFLAGS", "-I#{MacOS.sdk_path}/usr/include/ffi" if MacOS.sdk_path_if_needed
-
-    virtualenv_install_with_resources
+    (buildpath/"src/github.com/ooni/probe-cli").install buildpath.children
 
-    man1.install Dir["data/*.1"]
-    pkgshare.install Dir["data/*"]
+    cd "src/github.com/ooni/probe-cli" do
+      system "./build.sh macos"
+      bin.install "CLI/darwin/amd64/ooniprobe"
+    end
+    ooni_home = Pathname.new "#{var}/ooniprobe"
+    ooni_home.mkpath
   end
 
   def post_install
-    return if (pkgshare/"decks-available").exist?
-
-    ln_s pkgshare/"decks", pkgshare/"decks-available"
-    ln_s pkgshare/"decks/web.yaml", pkgshare/"current.deck"
+    ENV["OONI_HOME"] = "#{HOMEBREW_PREFIX}/var/ooniprobe"
+    system bin/"ooniprobe", "onboard", "--yes"
   end
 
   def caveats
     <<~EOS
-      Decks are installed to #{opt_pkgshare}/decks.
+      By enabling the homebrew service you will not be shown the informed consent.
```

Comment:
Why not actually?

Reply:
Because it's running in the background and there is no way to show a prompt to users.

Reply:
If it doesn't run without consent then that's fine too. I don't think skipping consent is a good price to pay for an easier first run.

I get the impression ooniprobe should always be ran as a service, so are you skipping the consent for all other install methods? Because then it might be better to remove it altogether.